### PR TITLE
Remove code needed to annotate IDL definitions in HTML spec

### PR DESCRIPTION
**Pull request by dontcallmedom**
Upstreamed in https://github.com/whatwg/html/pull/6762

as usual, not to be merged until upstream PR is merged, and comparison of extracted data is verified

### CODE REVIEW

**Comment by dontcallmedom** [comment]
we should be good to go after whatwg/html#6776 gets merged

**Comment by dontcallmedom** [comment]
good to go from my perspective

**Review by tidoust — APPROVED** [review]

**Pull request merged**
